tempto-examples/src/main/resources/sql-tests/testcases/tpcds/select_star_from_customer.sql

```diff
@@ -0,0 +1,6 @@
+-- database: presto_tpcds; groups: tpcds; tables: tpcds.customer
```

Comment:
Name of the file does not correspond to what's in the file.
`select_count_from_customer.sql` would be more appropriate.